modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBarUser.vue

```diff
@@ -54,7 +54,7 @@ export default {
       return {
         label: this.user.label || '',
         icon: 'chevron-down-icon',
-        modifiers: [ 'icon-right' ],
+        modifiers: [ 'icon-right', 'no-motion' ],
```

Comment:
`no-motion` removes hover/focus movement and was generalized from an existing type, this is just an update

Reply:
Idea: use `reduced-motion` to reflect `prefers-reduced-motion`? Just an idea in case it could be connected later. https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion